### ISSUE TITLE
Server listen does not clean up used socket path

### ISSUE THREAD

**Issue by Downchuck** [sidorares/osquery-node#3]
Apparently this can lead to many "/var/osquery/osquery.em.*" files and attempts address in use errors. This may only happen if the script is terminated unexpectedly (not sure).

**Comment by sidorares** [comment]
not sure if there is automatic way to delete them, I guess you need to unlink explicitly when no longer need

**Comment by Downchuck** [comment]
Correct; http://stackoverflow.com/questions/16178239/gracefully-shutdown-unix-socket-server-on-nodejs-running-under-forever
